js/widgets/fixedToolbar.js

```diff
@@ -43,10 +43,11 @@
 			transition: "slide", //can be none, fade, slide (slide maps to slideup or slidedown)
 			fullscreen: false,
 			tapToggle: true,
```

Comment:
@gabrielschulhof @cgack @jaspermdegroot @agcolom Thoughts on switching this to false by default and making it an extension? I feel like this is atypical behavior these days and personally i never liked it any time i used fixed toolbars i always turned it off?